frontend/src/pages/files/FileControlsWrapper.tsx

```diff
@@ -73,9 +74,120 @@ const FileManager = observer((props: FileManagerProps) => {
 
     let securityNotice = <></>;
 
-    // TODO: There is a lot of duplicated code here. This can be refactored to be more readable and maintanable.
     if (path.type === 'mine') {
-      if (props.cursor.isSearchable()) {
+      // This is the section of the left bar where the user can edit the properties of the dataset
+      const getEditableDescriptionSection = (searchJson: ReactiveJsonFile) => {
+        return <>
+          <div>
+            <label className="mb-1">
+              Dataset Title:
+              <OverlayTrigger
+                placement="right"
+                delay={{ show: 50, hide: 400 }}
+                overlay={(props) => (
+                  <Tooltip id="button-tooltip" {...props}>
+                    This is the name of the dataset, currently not editable
+                  </Tooltip>
+                )}>
+                <i className="mdi mdi-help-circle-outline text-muted vertical-middle" style={{ marginLeft: '5px' }}></i>
+              </OverlayTrigger>
+            </label>
+            <textarea className="mb-2" style={{ width: '100%' }} placeholder="Full title of dataset" value={searchJson.getAttribute("title", "")} onChange={(e) => {
```

Comment:
Should we use here a textarea, or a regular input? I understand the name of the dataset can be pretty long, so maybe a textarea is good here.